app/models/recipe.rb

```diff
@@ -10,6 +10,7 @@ class Recipe < ApplicationRecord
   # == Validations ==========================================================
 
   validates :name, presence: true
+  validates :price, allow_blank: true, numericality: { greater_than: 0 }
```

Comment:
```suggestion
  validates :price, numericality: { greater_than: 0 }, allow_blank: true
```
it was ok but this order is common :D